### PR TITLE
chore: update rollup/deps and deno version in ci test

### PR DESCRIPTION
**Pull request by buttercubz**

### CODE REVIEW

**Comment by cmorten** [comment]
Hey @buttercubz 👋 

Thanks for the contribution! Been on hols without laptop so not been able to whip round and get the latest versions updated in all my modules 😅 thanks for doing some of the chore work!

I run a `fmt` check in CI, if you run and commit the result of `make fmt` that should cover off the failures.

File: src/rollup/rollup.ts [hunk 1]
```diff
@@ -44,7 +44,8 @@ export async function rollup(
   };
 
   try {
-    const bundle = await _rollup(options);
+    // deno-lint-ignore no-explicit-any
+    const bundle = await _rollup(options as any);
```
Comment (cmorten) [inline src/rollup/rollup.ts:48]: Guessing seeing a type mismatch? Check out some of the extended types around https://github.com/cmorten/deno-rollup/blob/main/src/rollup/mod.ts#L165, suspect may have to add the new `sanitizeFileName` or sim. somewhere, shouldn’t need to cast as `any`.

Comment (buttercubz) [reply src/rollup/rollup.ts]: i found the problem, now this is fixed

**Comment by cmorten** [comment]
Superseded by #34

**Pull request closed without merging**